Model/features.php

```diff
@@ -0,0 +1,220 @@
+    }
+  }
+
+  /**
+   * Checks Mail Id validation with mailBoxLayer API.
+   * 
+   * @param  $mailId
+   *  Stores the Mail Id of the user. 
+   * 
+   **/
+  function validMailBox(string $mailId)
+  {
+
+    // API Calling using HttpGuzzle.
+    $client = new Client([
+      // Base uri of the site
+      'base_uri' => 'https://api.apilayer.com/ ?email=',
+    ]);
+
+    $request = $client->request('GET', 'email_verification/check', [
+      "headers" => [
+        'apikey' => 'H2AIxxMvhiT1uUKhxs7TuSMJmysHASNI'
+      ],
+      'query' => [
+        'email' => $mailId,
+      ]
+    ]);
+    $response = $request->getBody();
+
+
+
+    // Checking format, mx, smtp, and deliverablity score for the mail
+    if (json_decode($response)->format_valid == TRUE && json_decode($response)->mx_found == TRUE && json_decode($response)->smtp_check == TRUE) {
+      echo "<br>(E-mail deliverablity score is: " . ((json_decode($response)->score) * 100) . "% ).";
+      return TRUE;
+    } 
+    else {
+      echo "<div class='error'>Error:<br>";
+
+      if (isset(json_decode($response)->format_valid) && json_decode($response)->format_valid == FALSE) {
+        echo "E-mail format is not valid<br>";
+      }
+      if (isset(json_decode($response)->mx_found) && json_decode($response)->mx_found == FALSE) {
+        echo "MX-Records not found<br>";
+      }
+      if (isset(json_decode($response)->smtp_check) && json_decode($response)->smtp_check == FALSE) {
+        echo "SMTP validation failed<br>";
+      }
+      echo "</div>";
+      return false;
+    }
+  }
+
+  /** 
+   * Send Mails using PHP-Mailer. 
+   * @param  $mailId
+   *   Takes mailId as input field data of the user. 
+   * 
+   **/
+  function sendMail(string $mailId, string $subject = "Subject", string $body = "no data found")
+  {
+    $mail = new PHPMailer(true);
+
+    try {
+      $mail->SMTPDebug = 0;
+      $mail->isSMTP();
+      $mail->Host = 'smtp.gmail.com';
+      $mail->SMTPAuth = true;
+      $mail->Username = $_ENV['SMTPMail'];
+      $mail->Password = $_ENV['SMTPKey'];
+      $mail->SMTPSecure = 'tls';
+      $mail->Port = 587;
+
+      $mail->setFrom($mailId, 'PostIt');
+      $mail->addAddress($mailId);
+
+      $mail->isHTML(true);
+      $mail->Subject = $subject;
+      $mail->Body = $body;
+      $mail->AltBody = 'Body in plain text for non-HTML mail clients';
+      $mail->send();
+      echo "<div class='success'>Mail has been sent successfully!</div>";
+    } 
+    catch (Exception $e) {
+      echo "Message could not be sent. Mailer Error: {$mail->ErrorInfo}";
+    }
+  }
+
+  /** 
+   * Get response body from url using Guzzle.
+   * @param  $url
+   *   Takes url as input and return response body. 
+   * @return Psr\Http\Message\StreamInterface
+   **/
+  function getURL(string $url)
+  {
+    $client = new Client([
+      // Base uri of the site
+      'base_uri' => $url,
+    ]);
+    $request = $client->request('GET');
+    return $request->getBody();
+  }
+
+}
+
```

Comment:
Mention parameter datatype in comments also, return type. Remove unnecessary single line comments.